ocp_resources/ocs_initialization.py

```diff
@@ -0,0 +1,49 @@
+from ocp_resources.constants import TIMEOUT_4MINUTES
+from ocp_resources.resource import NamespacedResource
+
+
+class OCSInitialization(NamespacedResource):
+    """
+    OCSInitialization object.
+
+    API reference (source code linked here due to lack of API doc for this resource):
+        https://github.com/red-hat-storage/ocs-operator/blob/main/api/v1/ocsinitialization_types.go
+
+    """
+
+    api_group = NamespacedResource.ApiGroup.OCS_OPENSHIFT_IO
+
+    def __init__(
+        self,
+        name=None,
+        namespace=None,
+        client=None,
+        teardown=False,
+        privileged_client=None,
+        yaml_file=None,
+        delete_timeout=TIMEOUT_4MINUTES,
+        enable_ceph_tools=False,
```

Comment:
Is False is the default?
If not, please set it to None and raise if the user didn't send it.

Reply:
Done